tests/Common/Common.h

```diff
@@ -825,30 +825,6 @@ DLL_API void va_listFunction(va_list v);
 struct DLL_API TestNestedTypes
 {
 public:
-    struct
```

Comment:
Move this to the new header, and comment it out with an explanation that we can't handle such code yet.

Reply:
I actually like this solution.